packages/client/hmi-client/src/components/articles/Document.vue

```diff
@@ -253,8 +253,13 @@ const formatCitation = (obj: { [key: string]: string }) => {
 	let citation: string;
 	if (Object.keys(obj).length <= 1) {
 		citation = obj.unstructured_citation ?? '';
+		citation = citation.replace(
+			/\bhttps?:\/\/\S+/,
+			`<a style='color: var(--text-color-subdued);' href=$&>$&</a>`
```

Comment:
Really neat, I like when RegEx are nicely implemented.

It will be nicer to not add styling within the JS, and just add a class, or target the link within the AccordionTab directly.
```suggestion
			`<a href="$&">$&</a>`
```

Reply:
I agree about styling, but that styling is already present in the css for this component and did not seem to get applied here (maybe because the html is dynamically generated?), so I had to add it inline.

Reply:
You might have to add a `>>> a`  selector somewhere.